lib/internal/bootstrap/node.js

```diff
@@ -156,13 +157,32 @@ const rawMethods = internalBinding('process_methods');
   process._getActiveHandles = rawMethods._getActiveHandles;
 
   process.getActiveResourcesInfo = function() {
+    var timeoutCount = 0;
+    ArrayPrototypeForEach(ObjectValues(internalTimers.timerListMap),
```

Comment:
IIRC a classic `for` loop is still the most performant way of iterating over an array.

Reply:
Changed it to a normal for loop.